src/main/resources/swagger.api/schemas/holding.json

```diff
@@ -10,6 +10,10 @@
     "permanentLocationId": {
       "type": "string",
       "description": "The permanent shelving location in which an item resides."
+    },
+    "discoverySuppress": {
+      "type": "boolean",
+      "description": "records the fact that the record should not be displayed in a discovery system"
```

Comment:
Maybe change `records the fact` to `Indicates`?

Reply:
Changed.